Adafruit_FeatherOLED.h

```diff
@@ -5,7 +5,12 @@
 */
 /**************************************************************************/
 
+/// @cond DISABLE
```

Comment:
Remove this and the other disable as well. This`#ifndef` isn't affecting the doxygen run